test/index.js

```diff
@@ -175,6 +175,18 @@ describe('glob-watcher', function() {
     watcher.on('ready', changeFile);
   });
 
+  it('does not emit an error (and crash) when no handlers attached', function(done) {
+    var expectedError = new Error('boom');
+
+    watcher = watch(outGlob, function(cb) {
+      cb(expectedError);
+      setTimeout(done, timeout * 3);
```

Comment:
just wondering, what is the 3 for, could it be a different number and affect the result?

Reply:
No, it's just a conservative number - the test below uses `timeout * 3` also.